<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtChat v.0.1</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: blue;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: white;
}

#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>QtChat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="chatPage">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTextEdit" name="roomTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="sayButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Say</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QListWidget" name="userListWidget"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="sayLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="archiveButton">
             <property name="text">
              <string>Archive</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="archivePage">
          <widget class="QPushButton" name="exitArchiveButton">
           <property name="geometry">
            <rect>
             <x>650</x>
             <y>490</y>
             <width>111</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="archiveTextEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>751</width>
             <height>461</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Archiwum rozm√≥w:</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="loginPage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>100</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>223</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QFrame" name="loginFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>255</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>170</red>
                   <green>255</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>0</red>
                   <green>255</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>170</red>
                   <green>255</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Button">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>170</red>
                   <green>255</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="Window">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>170</red>
                   <green>255</green>
                   <blue>127</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="spacing">
               <number>20</number>
              </property>
              <item row="1" column="1">
               <widget class="QLineEdit" name="userLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Nazwa u≈ºytkownika:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Nazwa hosta:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="serverLineEdit">
                <property name="text">
                 <string>localhost</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="loginButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Zaloguj</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>267</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>serverLineEdit</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>sayLineEdit</tabstop>
  <tabstop>sayButton</tabstop>
  <tabstop>roomTextEdit</tabstop>
  <tabstop>userListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sayLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sayButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userListWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>userListWidget</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
